bin/web

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+set -o errexit -o nounset -o pipefail
+
+ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )"/.. && pwd )"
+
+DEV_PORT=8080
+export NODE_ENV=${NODE_ENV:=development}
+PUBLIC_API_SOURCE=k8s
+
+function -h {
+cat <<USAGE
+USAGE: web <command>
+
+  * build - build the assets
+  * dev - run a dev server (with live reloading). Options:
+      -p $DEV_PORT
+  * port-forward - forward the controller from a running k8s cluster to localhost
+  * run - run a local server (sans. reloading)
+  * setup - get the environment setup for development
+      Note: any command line options are passed on to yarn
+  * test - run the tests
+      Note: any command line options are passed on to karma
+USAGE
+}; function --help { -h ;}
+
+function dev {
+  setup
+
+  while getopts "p:" opt; do
+    case "$opt" in
+      p) DEV_PORT=$OPTARG
+    esac
+  done
+
+  nc -z localhost 8085 || port-forward &
```

Comment:
This is super handy for the `dev` target. I think it would also be useful to add it to the `run` target below.